lib/dpl/provider/releases.rb

```diff
@@ -100,27 +100,36 @@ def push_app
         end
 
         files.each do |file|
-          already_exists = false
+          existing_url = nil
           filename = Pathname.new(file).basename.to_s
           api.release(release_url).rels[:assets].get.data.each do |existing_file|
             if existing_file.name == filename
-              already_exists = true
+              existing_url = existing_file.url
             end
           end
-          if already_exists
-            log "#{filename} already exists, skipping."
+          if !existing_url
+            log "#{filename} doesn't exist, uploading."
```

Comment:
Is this desirable? It seems to me that this is the default behavior, and this would be mostly noise.